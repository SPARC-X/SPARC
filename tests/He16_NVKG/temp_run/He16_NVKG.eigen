Final eigenvalues and occupation numbers

kred #1 = (0.000000,0.000000,0.000000)
n        eigval                 occ
1       -6.560144937679E-01     2.000000000000
2       -5.614712340418E-01     2.000000000000
3       -5.558449979137E-01     2.000000000000
4       -4.088810273370E-01     2.000000000000
5       -4.080248197764E-01     2.000000000000
6       -4.061154387142E-01     2.000000000000
7       -3.341458499515E-01     2.000000000000
8       -3.287864967030E-01     2.000000000000
9       -3.271921502478E-01     2.000000000000
10      -3.214155464474E-01     2.000000000000
11      -2.229621245860E-01     2.000000000000
12      -2.216727544273E-01     2.000000000000
13      -2.203027525635E-01     2.000000000000
14      -1.618182686834E-01     2.000000000000
15      -1.538700566630E-01     2.000000000000
16      -8.263969594226E-02     2.000000000000
17       2.903778948429E-01     0.000000064454
18       2.931060486970E-01     0.000000021957
19       2.999167487779E-01     0.000000001493
20       4.183061880040E-01     0.000000000000
21       4.204797246119E-01     0.000000000000
22       4.225990683895E-01     0.000000000000
23       4.257289113639E-01     0.000000000000
24       6.233871861609E-01     0.000000000000
