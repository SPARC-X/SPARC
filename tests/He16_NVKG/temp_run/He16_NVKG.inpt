# nprocs: 8



CELL: 12.151112185593 6.0755560927965 6.0755560927965  
LATVEC:                       
  1 0 0
  0 1 0
  0 0 1
BC: P P P                     


EXCHANGE_CORRELATION: LDA_PZ  


NSTATES: 24                   
     
ELEC_TEMP_TYPE: fermi-dirac   
ELEC_TEMP: 800               



MESH_SPACING: 0.08          
FD_ORDER: 12                 
KPOINT_GRID: 1 1 1           
KPOINT_SHIFT: 0.0 0.0 0.0     

MIXING_VARIABLE: potential    

TOL_SCF: 1e-6                 # density/potential relative residual convergence (unitless)

MAXIT_SCF: 100                 # max SCF iterations
MINIT_SCF: 3                  # min SCF iterations


# MD ##############################################
MD_FLAG: 1                  # 1= MD, 0= no MD (default)
ION_TEMP: 800                 # kelvin
ION_TEMP_END: 800             # kelvin: vary temp linearly from ION_TEMP to ION_TEMP_END
      
MD_METHOD: NVK_G              # NVE, NVT_NH (Nose-Hoover), NVK_G (Gaussian)
#QMASS: 10                     # mass for NH thermostat (au): default ~1654
MD_TIMESTEP: 0.6              # fs
MD_NSTEP: 10                   # run MD_NSTEP steps or TWTIME minutes, whichever comes first
#TWTIME: 1400
ION_VEL_DSTR: 1               # initial velocity distrib: 1= uniform, 2= Maxwell-Boltzmann (default)
ION_VEL_DSTR_RAND: 0          # initial velocity distrib seed: 0= fixed (default), 1= random
PRINT_RESTART: 1              # output .restart files for restarting
PRINT_RESTART_FQ: 3           # .restart files output every PRINT_RESTART_FQ steps
RESTART_FLAG: 1               # 1= restart MD from .restart file if present, 0= start new

# outputs #########################################
CALC_PRES: 1
CALC_STRESS: 1
PRINT_ATOMS: 1
PRINT_VELS: 1
PRINT_FORCES: 1
PRINT_EIGEN: 1                # print eigenvalues and occupations to .eigen file
PRINT_DENSITY: 0
PRINT_MDOUT: 1                # print MD output to .aimd file
