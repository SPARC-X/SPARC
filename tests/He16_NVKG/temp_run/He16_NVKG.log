
error = 3.69e-08 > TOL_PSEUDOCHARGE = 1.00e-08

WARNING: upperbond for pseudocharge radius (Rbmax_?) is not big enough! Rbmax_x = 6.3, Rbmax_y = 6.3, Rbmax_z = 6.3

The program took 717.034 s.
