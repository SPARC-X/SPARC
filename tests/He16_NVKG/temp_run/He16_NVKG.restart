:MDSTEP: 10
:R(Bohr):
  1.2021393836E+01   6.0306748310E+00   1.0942060394E-01
  2.5376539275E-02   2.0677487299E-01   2.9941601205E+00
  1.2146291089E+01   2.8971356605E+00   1.0410108261E-01
  1.2101390366E+01   2.9431363989E+00   3.1492949780E+00
  3.0293716459E+00   5.9109828739E+00   1.1066118964E-01
  2.9199026723E+00   9.3904373916E-04   3.2047789037E+00
  3.0286725512E+00   2.8808052486E+00   9.2683703714E-02
  3.1945299710E+00   3.0443352470E+00   2.8709906129E+00
  6.3551101098E+00   6.0754946630E+00   6.0129945905E+00
  6.0483563490E+00   1.2884647264E-01   2.8358334448E+00
  6.0256880436E+00   3.3316630195E+00   4.5630338490E-02
  6.1112561484E+00   3.1720858161E+00   3.2290547120E+00
  9.0721699089E+00   2.9155165424E-04   5.9043519081E+00
  8.8662061853E+00   6.0721458018E+00   3.0386758713E+00
  9.2545338016E+00   2.9039745121E+00   5.9939358617E+00
  9.1597604529E+00   3.0051627291E+00   2.8323247278E+00
:V(Bohr/atu):
 -5.8015832932E-04  -4.5221624225E-04   3.2511938050E-04
  1.9808968540E-05   7.2327212053E-04  -1.5565610338E-04
  1.3493478393E-04  -5.2730284741E-04   5.0042993460E-04
 -1.3430854928E-04  -2.2667944396E-05   6.4969045196E-04
  8.8128221977E-05  -8.2830987879E-04   5.8464347641E-04
 -5.4354383639E-04   7.0157084640E-05   9.2287756921E-04
 -1.1994820722E-04  -8.5049890469E-04   6.4218467295E-05
  7.2333752560E-04   1.1919889895E-04  -6.8617174272E-04
  1.1368642322E-03   2.9367063074E-04  -5.0343447885E-04
 -4.7721981859E-04   6.6891175966E-04  -1.0228106699E-03
 -3.1456011663E-04   1.1099330855E-03   4.5948671998E-04
  3.0738869686E-04   5.1373780024E-04   9.8698796197E-04
  3.1426995572E-05  -1.4782510535E-04  -6.3345250839E-04
 -1.0299100480E-03  -8.9280944159E-05  -3.1356867716E-05
  5.3677199454E-04  -5.1960233502E-04  -4.9984609244E-04
  2.2098748626E-04  -6.1177178182E-05  -9.6072549853E-04
:TEL(K): 800
:TIO(K): 800.000000000001
