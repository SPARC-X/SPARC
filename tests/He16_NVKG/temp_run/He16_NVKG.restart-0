:MDSTEP: 9
:R(Bohr):
  1.2035925272E+01   6.0410038144E+00   1.0063298902E-01
  2.4529205752E-02   1.8795484824E-01   2.9981310110E+00
  1.2143528580E+01   2.9106407409E+00   9.1799749653E-02
  1.2105070195E+01   2.9452989252E+00   3.1336723427E+00
  3.0276613244E+00   5.9313037590E+00   9.6465826072E-02
  2.9334173500E+00   6.0750309179E+00   3.1823944212E+00
  3.0313699221E+00   2.9014476072E+00   8.9649798919E-02
  3.1765793747E+00   3.0417476474E+00   2.8883406082E+00
  6.3262141641E+00   6.0692859113E+00   6.0246562292E+00
  6.0588453763E+00   1.1244612233E-01   2.8608938518E+00
  6.0331205161E+00   3.3031352770E+00   3.5220283046E-02
  6.1041977947E+00   3.1589194915E+00   3.2049271959E+00
  9.0723395004E+00   3.4013684823E-03   5.9207362168E+00
  8.8921997397E+00   6.0741151424E+00   3.0393528708E+00
  9.2407199337E+00   2.9172666642E+00   6.0058542894E+00
  9.1542914210E+00   3.0070065976E+00   2.8561649661E+00
:V(Bohr/atu):
 -5.8907602428E-04  -3.8213546152E-04   3.8008792709E-04
  4.7479197050E-05   7.9095961679E-04  -1.6524938238E-04
  8.9174826271E-05  -5.6055188931E-04   4.9201930562E-04
 -1.6136082987E-04  -1.4633250374E-04   6.1052553379E-04
  5.1184260941E-05  -8.0946066445E-04   5.6088285633E-04
 -5.4494906878E-04   4.9222202783E-05   8.8138234403E-04
 -9.7992498263E-05  -8.1405348140E-04   1.7479370074E-04
  7.2305197606E-04   9.1124311236E-05  -7.1096589638E-04
  1.1876487134E-03   2.0916830256E-04  -4.3922729019E-04
 -3.7227203118E-04   6.5168947018E-04  -9.9709572919E-04
 -2.8638209594E-04   1.1851909265E-03   3.8331448369E-04
  2.6361566959E-04   5.4563768124E-04   9.5773458788E-04
 -4.0430937130E-05  -1.0440355174E-04  -6.8339884260E-04
 -1.0631700310E-03  -6.9531280624E-05  -2.2899624894E-05
  5.7385110516E-04  -5.4986541226E-04  -4.6241383570E-04
  2.1962776798E-04  -8.6658266233E-05  -9.5949013782E-04
:TEL(K): 800
:TIO(K): 800.000000000001
