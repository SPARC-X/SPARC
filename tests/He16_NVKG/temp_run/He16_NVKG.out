***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 21:00:57 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 12.151112185593 6.0755560927965 6.0755560927965 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 152 76 76
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
ELEC_TEMP: 800
CHEB_DEGREE: 54
NSTATES: 24
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 1
TWTIME: 1E+09
MD_FLAG: 1
MD_METHOD: NVK_G
MD_TIMESTEP: 0.6
MD_NSTEP: 10
ION_VEL_DSTR: 1
ION_VEL_DSTR_RAND: 0
ION_TEMP: 800
QMASS: 1653.65493345972
RESTART_FLAG: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: kerker
TOL_PRECOND: 6.39E-06
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
PRINT_MDOUT: 1
PRINT_VELS: 1
PRINT_RESTART: 1
PRINT_RESTART_FQ: 3
OUTPUT_FILE: He16_NVKG
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 8
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 2 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  8
Mesh spacing                       :  0.0799415 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  He16_NVKG.out
Final eigenvalues printed to       :  He16_NVKG.eigen
MD output printed to               :  He16_NVKG.aimd
Total number of atom types         :  1
Total number of atoms              :  16
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  He 2
Pseudopotential                    :  He-2-0.5-loc.psp8
Atomic mass                        :  4.002602
Pseudocharge radii of atom type 1  :  6.32 6.32 6.32 (x, y, z dir)
Number of atoms of type 1          :  16
Estimated total memory usage       :  1.13 GB
Estimated memory per processor     :  144.01 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7565828870E+00        2.954E-02        17.257
2            -2.7776240992E+00        1.385E-02        7.078
3            -2.7786245931E+00        9.898E-03        6.702
4            -2.7787603155E+00        4.339E-03        6.505
5            -2.7787779541E+00        1.431E-03        6.396
6            -2.7787781090E+00        9.582E-04        5.849
7            -2.7787781489E+00        4.256E-04        5.988
8            -2.7787781743E+00        1.125E-04        5.892
9            -2.7787781751E+00        6.811E-05        5.596
10           -2.7787781560E+00        3.162E-05        5.572
11           -2.7787781785E+00        2.062E-05        5.324
12           -2.7787781830E+00        1.569E-05        5.092
13           -2.7787781922E+00        1.062E-05        5.396
14           -2.7787781832E+00        3.758E-06        4.965
15           -2.7787781860E+00        1.330E-06        4.838
16           -2.7787781924E+00        5.327E-07        3.610
Total number of SCF: 16    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787781924E+00 (Ha/atom)
Total free energy                  : -4.4460451078E+01 (Ha)
Band structure energy              : -1.0686323703E+01 (Ha)
Exchange correlation energy        : -1.6847455607E+01 (Ha)
Self and correction energy         : -2.9007561503E+02 (Ha)
Entropy*kb*T                       : -2.3157329698E-09 (Ha)
Fermi level                        :  2.4078439506E-01 (Ha)
RMS force                          :  3.8651111513E-07 (Ha/Bohr)
Maximum force                      :  7.4533514982E-07 (Ha/Bohr)
Time for force calculation         :  0.749 (sec)
Pressure                           :  1.3429891894E+02 (GPa)
Maximum stress                     :  1.6845953248E+02 (GPa)
Time for stress calculation        :  2.363 (sec)
MD step time                       :  108.386 (sec)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7785579863E+00        2.430E-02        7.529
2            -2.7787327585E+00        1.860E-02        6.608
3            -2.7788023463E+00        5.619E-03        6.617
4            -2.7787985281E+00        3.681E-03        6.436
5            -2.7788048651E+00        6.783E-04        6.049
6            -2.7788049510E+00        4.569E-04        6.084
7            -2.7788050242E+00        6.271E-05        5.788
8            -2.7788050131E+00        5.445E-05        5.459
9            -2.7788050260E+00        1.214E-05        5.356
10           -2.7788050269E+00        4.871E-06        4.941
11           -2.7788050257E+00        4.892E-06        4.880
12           -2.7788050280E+00        3.017E-06        4.729
13           -2.7788050252E+00        2.025E-06        4.494
14           -2.7788050278E+00        2.055E-06        4.590
15           -2.7788050277E+00        1.184E-06        4.656
16           -2.7788050261E+00        1.127E-06        4.569
17           -2.7788050276E+00        5.311E-07        3.382
Total number of SCF: 17    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788050276E+00 (Ha/atom)
Total free energy                  : -4.4460880442E+01 (Ha)
Band structure energy              : -1.0687166179E+01 (Ha)
Exchange correlation energy        : -1.6847547587E+01 (Ha)
Self and correction energy         : -2.9008286487E+02 (Ha)
Entropy*kb*T                       : -4.2387766386E-09 (Ha)
Fermi level                        :  2.4246920361E-01 (Ha)
RMS force                          :  2.8435021790E-03 (Ha/Bohr)
Maximum force                      :  4.9120330314E-03 (Ha/Bohr)
Time for force calculation         :  0.743 (sec)
Pressure                           :  1.3428287993E+02 (GPa)
Maximum stress                     :  1.6845151911E+02 (GPa)
Time for stress calculation        :  2.373 (sec)
MD step time                       :  98.471 (sec)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7785498154E+00        2.399E-02        7.552
2            -2.7787225793E+00        1.835E-02        6.627
3            -2.7787908663E+00        5.532E-03        6.615
4            -2.7787871187E+00        3.632E-03        6.445
5            -2.7787933092E+00        6.707E-04        6.045
6            -2.7787933870E+00        4.500E-04        6.126
7            -2.7787934611E+00        6.278E-05        5.790
8            -2.7787934475E+00        5.157E-05        5.437
9            -2.7787934658E+00        1.179E-05        5.366
10           -2.7787934619E+00        4.239E-06        4.865
11           -2.7787934620E+00        4.440E-06        4.830
12           -2.7787934643E+00        2.497E-06        4.966
13           -2.7787934613E+00        1.555E-06        4.686
14           -2.7787934617E+00        1.325E-06        4.560
15           -2.7787934630E+00        1.562E-06        4.741
16           -2.7787934643E+00        9.740E-07        3.368
Total number of SCF: 16    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787934643E+00 (Ha/atom)
Total free energy                  : -4.4460695428E+01 (Ha)
Band structure energy              : -1.0688793622E+01 (Ha)
Exchange correlation energy        : -1.6847573749E+01 (Ha)
Self and correction energy         : -2.9009059920E+02 (Ha)
Entropy*kb*T                       : -4.2475759631E-09 (Ha)
Fermi level                        :  2.4267553111E-01 (Ha)
RMS force                          :  5.6865119018E-03 (Ha/Bohr)
Maximum force                      :  9.8174725052E-03 (Ha/Bohr)
Time for force calculation         :  0.743 (sec)
Pressure                           :  1.3426445888E+02 (GPa)
Maximum stress                     :  1.6846352445E+02 (GPa)
Time for stress calculation        :  2.387 (sec)
MD step time                       :  94.337 (sec)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787037128E+00        4.098E-04        6.171
2            -2.7787596327E+00        2.382E-04        6.029
3            -2.7787607082E+00        9.083E-05        5.861
4            -2.7787607317E+00        8.558E-05        5.201
5            -2.7787607390E+00        3.830E-05        5.461
6            -2.7787607071E+00        1.803E-05        5.282
7            -2.7787607401E+00        5.929E-06        5.132
8            -2.7787607374E+00        6.516E-06        4.576
9            -2.7787607353E+00        2.821E-06        4.982
10           -2.7787607505E+00        7.621E-07        3.522
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787607505E+00 (Ha/atom)
Total free energy                  : -4.4460172007E+01 (Ha)
Band structure energy              : -1.0691426696E+01 (Ha)
Exchange correlation energy        : -1.6847487528E+01 (Ha)
Self and correction energy         : -2.9009095188E+02 (Ha)
Entropy*kb*T                       : -4.2285846986E-09 (Ha)
Fermi level                        :  2.4299023250E-01 (Ha)
RMS force                          :  8.5274078319E-03 (Ha/Bohr)
Maximum force                      :  1.4715314122E-02 (Ha/Bohr)
Time for force calculation         :  0.742 (sec)
Pressure                           :  1.3422873832E+02 (GPa)
Maximum stress                     :  1.6848092868E+02 (GPa)
Time for stress calculation        :  2.379 (sec)
MD step time                       :  58.302 (sec)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7786757067E+00        4.011E-04        6.195
2            -2.7787316046E+00        2.270E-04        6.037
3            -2.7787326764E+00        7.218E-05        5.866
4            -2.7787326967E+00        6.846E-05        5.157
5            -2.7787327128E+00        3.862E-05        5.118
6            -2.7787326964E+00        1.456E-05        5.169
7            -2.7787326982E+00        4.359E-06        5.188
8            -2.7787326979E+00        3.375E-06        4.685
9            -2.7787326972E+00        1.622E-06        4.981
10           -2.7787326988E+00        9.742E-07        3.452
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787326988E+00 (Ha/atom)
Total free energy                  : -4.4459723181E+01 (Ha)
Band structure energy              : -1.0695400663E+01 (Ha)
Exchange correlation energy        : -1.6847226492E+01 (Ha)
Self and correction energy         : -2.9008710454E+02 (Ha)
Entropy*kb*T                       : -4.2237686507E-09 (Ha)
Fermi level                        :  2.4342998910E-01 (Ha)
RMS force                          :  1.1357533935E-02 (Ha/Bohr)
Maximum force                      :  1.9586744661E-02 (Ha/Bohr)
Time for force calculation         :  0.744 (sec)
Pressure                           :  1.3415886094E+02 (GPa)
Maximum stress                     :  1.6848796633E+02 (GPa)
Time for stress calculation        :  2.367 (sec)
MD step time                       :  57.923 (sec)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7786503744E+00        3.865E-04        6.202
2            -2.7787062249E+00        2.203E-04        6.010
3            -2.7787072948E+00        7.072E-05        5.810
4            -2.7787073141E+00        6.782E-05        5.062
5            -2.7787073168E+00        4.939E-05        5.042
6            -2.7787073253E+00        1.824E-05        5.016
7            -2.7787073154E+00        1.075E-05        4.998
8            -2.7787073149E+00        3.696E-06        4.948
9            -2.7787073111E+00        1.797E-06        4.850
10           -2.7787073122E+00        1.100E-06        4.825
11           -2.7787073054E+00        6.480E-07        3.389
Total number of SCF: 11    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787073054E+00 (Ha/atom)
Total free energy                  : -4.4459316887E+01 (Ha)
Band structure energy              : -1.0700805631E+01 (Ha)
Exchange correlation energy        : -1.6846620722E+01 (Ha)
Self and correction energy         : -2.9008553868E+02 (Ha)
Entropy*kb*T                       : -4.2220842537E-09 (Ha)
Fermi level                        :  2.4396964622E-01 (Ha)
RMS force                          :  1.4163067518E-02 (Ha/Bohr)
Maximum force                      :  2.4398122325E-02 (Ha/Bohr)
Time for force calculation         :  0.743 (sec)
Pressure                           :  1.3403649500E+02 (GPa)
Maximum stress                     :  1.6846518625E+02 (GPa)
Time for stress calculation        :  2.374 (sec)
MD step time                       :  62.239 (sec)
===================================================================
                    Self Consistent Field (SCF#7)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7786532317E+00        3.710E-04        6.147
2            -2.7787090172E+00        2.119E-04        6.057
3            -2.7787100838E+00        7.099E-05        5.802
4            -2.7787101056E+00        6.781E-05        5.093
5            -2.7787101058E+00        5.155E-05        4.978
6            -2.7787101145E+00        2.361E-05        5.101
7            -2.7787101069E+00        1.029E-05        5.216
8            -2.7787101085E+00        2.827E-06        4.914
9            -2.7787101052E+00        1.665E-06        4.683
10           -2.7787101054E+00        1.088E-06        4.438
11           -2.7787101098E+00        6.883E-07        3.393
Total number of SCF: 11    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787101098E+00 (Ha/atom)
Total free energy                  : -4.4459361757E+01 (Ha)
Band structure energy              : -1.0708005085E+01 (Ha)
Exchange correlation energy        : -1.6845514727E+01 (Ha)
Self and correction energy         : -2.9008861293E+02 (Ha)
Entropy*kb*T                       : -4.2027042698E-09 (Ha)
Fermi level                        :  2.4457566644E-01 (Ha)
RMS force                          :  1.6921378279E-02 (Ha/Bohr)
Maximum force                      :  2.9098227365E-02 (Ha/Bohr)
Time for force calculation         :  0.745 (sec)
Pressure                           :  1.3383430732E+02 (GPa)
Maximum stress                     :  1.6839033102E+02 (GPa)
Time for stress calculation        :  2.390 (sec)
MD step time                       :  61.920 (sec)
===================================================================
                    Self Consistent Field (SCF#8)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787005591E+00        3.642E-04        6.138
2            -2.7787562974E+00        1.944E-04        6.056
3            -2.7787573498E+00        6.593E-05        5.777
4            -2.7787573711E+00        6.190E-05        5.003
5            -2.7787573714E+00        4.151E-05        4.989
6            -2.7787573664E+00        1.847E-05        4.847
7            -2.7787573763E+00        1.095E-05        4.838
8            -2.7787573742E+00        6.118E-06        4.757
9            -2.7787573778E+00        2.345E-06        4.875
10           -2.7787573661E+00        7.582E-07        3.580
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7787573661E+00 (Ha/atom)
Total free energy                  : -4.4460117857E+01 (Ha)
Band structure energy              : -1.0717266001E+01 (Ha)
Exchange correlation energy        : -1.6843688285E+01 (Ha)
Self and correction energy         : -2.9009017080E+02 (Ha)
Entropy*kb*T                       : -4.1875974169E-09 (Ha)
Fermi level                        :  2.4524110724E-01 (Ha)
RMS force                          :  1.9599148007E-02 (Ha/Bohr)
Maximum force                      :  3.3611010048E-02 (Ha/Bohr)
Time for force calculation         :  0.748 (sec)
Pressure                           :  1.3352613974E+02 (GPa)
Maximum stress                     :  1.6823616960E+02 (GPa)
Time for stress calculation        :  2.381 (sec)
MD step time                       :  56.955 (sec)
===================================================================
                    Self Consistent Field (SCF#9)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7787833989E+00        3.508E-04        6.114
2            -2.7788390727E+00        1.766E-04        5.991
3            -2.7788401160E+00        6.776E-05        5.775
4            -2.7788401356E+00        6.371E-05        5.036
5            -2.7788401298E+00        5.276E-05        4.904
6            -2.7788401411E+00        1.930E-05        5.126
7            -2.7788401403E+00        1.164E-05        4.713
8            -2.7788401454E+00        3.635E-06        4.878
9            -2.7788401387E+00        2.707E-06        4.672
10           -2.7788401373E+00        7.297E-07        3.595
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7788401373E+00 (Ha/atom)
Total free energy                  : -4.4461442197E+01 (Ha)
Band structure energy              : -1.0728567151E+01 (Ha)
Exchange correlation energy        : -1.6840864566E+01 (Ha)
Self and correction energy         : -2.9009126358E+02 (Ha)
Entropy*kb*T                       : -4.1570778936E-09 (Ha)
Fermi level                        :  2.4593629396E-01 (Ha)
RMS force                          :  2.2158053045E-02 (Ha/Bohr)
Maximum force                      :  3.7844356808E-02 (Ha/Bohr)
Time for force calculation         :  0.747 (sec)
Pressure                           :  1.3308488619E+02 (GPa)
Maximum stress                     :  1.6797392951E+02 (GPa)
Time for stress calculation        :  2.400 (sec)
MD step time                       :  56.889 (sec)
===================================================================
                    Self Consistent Field (SCF#10)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.7789410023E+00        3.327E-04        6.106
2            -2.7789965321E+00        1.707E-04        5.998
3            -2.7789975662E+00        6.863E-05        5.790
4            -2.7789975845E+00        6.340E-05        5.016
5            -2.7789975825E+00        4.138E-05        4.977
6            -2.7789975873E+00        2.241E-05        4.811
7            -2.7789975930E+00        8.483E-06        5.139
8            -2.7789975917E+00        5.290E-06        4.668
9            -2.7789975873E+00        2.437E-06        4.763
10           -2.7789975839E+00        1.052E-06        4.836
11           -2.7789975858E+00        7.604E-07        3.381
Total number of SCF: 11    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000000000000.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.7789975858E+00 (Ha/atom)
Total free energy                  : -4.4463961372E+01 (Ha)
Band structure energy              : -1.0742315390E+01 (Ha)
Exchange correlation energy        : -1.6836902882E+01 (Ha)
Self and correction energy         : -2.9009216850E+02 (Ha)
Entropy*kb*T                       : -4.1385303589E-09 (Ha)
Fermi level                        :  2.4667472041E-01 (Ha)
RMS force                          :  2.4548759254E-02 (Ha/Bohr)
Maximum force                      :  4.1687567379E-02 (Ha/Bohr)
Time for force calculation         :  0.744 (sec)
Pressure                           :  1.3248253674E+02 (GPa)
Maximum stress                     :  1.6757151477E+02 (GPa)
Time for stress calculation        :  2.387 (sec)
MD step time                       :  61.550 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  717.034 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
